gazelle/resolve.go

```diff
@@ -54,15 +54,14 @@ func (l *swiftLang) Resolve(
 	}
 	externalModules := make([]string, 0, len(swiftImports))
 	for _, imp := range swiftImports {
-		if swift.IsBuiltInModule(imp) {
-			rr.AddBuiltin(imp)
-			continue
-		}
 		findResults := ix.FindRulesByImportWithConfig(
 			c, resolve.ImportSpec{Lang: swiftLangName, Imp: imp}, swiftLangName)
 		if len(findResults) > 0 {
 			addToDeps(findResults[0].Label)
 			rr.AddLocal(imp, findResults)
+		} else if swift.IsBuiltInModule(imp) {
+			rr.AddBuiltin(imp)
+			continue
```

Comment:
Im thinking we should move this out after external modules actually. Otherwise we might drop them if they are external but have the same name as a built in module

Reply:
I moved to after external product resolution